### DISCUSSION TITLE
What is the logic to combine these two frameworks? Hardhat + Foundry

### DISCUSSION THREAD

**Discussion by lawweiliang** [NomicFoundation/hardhat#3781] [General]
Hi Hardhat Community,

Why do we need to combine both hardhat and foundry into 1 project [^1]?
To make the project `look`  fancy or cool?

What is the main use case for combining this two?

For my understanding.
Hardhat, foundry, brownies and truffle only have 1 big use case. Simplify developer process in generating and deploying contracts to blockchain. That't it.  

[^1]: ]https://hardhat.org/hardhat-runner/plugins/nomicfoundation-hardhat-foundry

**Comment by alcuadrado** [comment]
They are different tools that are better suited for different things, plus there's always personal preferences.
